components/splash/splash_for_developers.js

```diff
@@ -35,10 +35,13 @@ const SplashForDevelopers = (props) => {
           <Subheading textAlign={{ base: 'center', lg: 'left' }}>
             Seamlessly give back to the maintainers you rely on
           </Subheading>
-          <Text marginBottom='1rem'>
+          <Text>
             Flossbank provides a <strong>free and frictionless</strong> way to
             give back to Open Source maintainers, especially those who don’t
-            have time for self promotion. (<TextLink text='How is that possible?' href='/how-it-works' />)
+            have time for self promotion.
+          </Text>
+          <Text marginBottom='1rem'>
+            (<TextLink text='How is that possible?' href='/how-it-works' />)
```

Comment:
Having these on new lines don’t really work great on medium sizing. (Turn phone horizontal)

Why did we move these to new lines?

Reply:
Not sure why, but for some reason when they were on the same line, the hover effect wasn't showing at all (even though it does for all other links).

Reply:
true but i dont think we should move them to a new line, makes it feel disjointed rather than part of the conversation.

let's try to figure out the hover when it's wrapped in a text or just leave it as is

Reply:
OK, changing it back and then we can work on figuring out why only these links are buggy! :)